program/src/state.rs

```diff
@@ -51,6 +51,7 @@ pub const MAX_NUM_IN_MARGIN_BASKET: u8 = 9;
 pub const INDEX_START: I80F48 = I80F48!(1_000_000);
 pub const PYTH_CONF_FILTER: I80F48 = I80F48!(0.10); // filter out pyth prices with conf > 10% of price
 pub const CENTIBPS_PER_UNIT: I80F48 = I80F48!(1_000_000);
+pub const REF_TIER_2_FACTOR: I80F48 = I80F48!(10);
```

Comment:
sad that this needs to be hardcoded, I guess making the share_2/surcharge_2 u16 is too limiting...

Reply:
Yes it's not ideal. I actually think u16 would be fine for the values that would realistically be used, just went with 32 for consistency. Wonder if I can change this now without breaking devnet...

Reply:
could set to 0 on devnet, change the program, re-set

Reply:
Tried this out, but unfortunately the MNGO required in native units will overflow a u32 as well. Since we have the quote decimals already on the group and the threshold is likely to be a large round number, we could store the 'UI' value and convert to native where required. It's a bit messy since tier 1 and tier 2 would then be in different units so I'm not sure I like this solution.

Reply:
Trying this out today, switching tier 2 params to u16 and storing the factor on the group as a u8